PeerioTechnologies/peerio-mobile
# chore: replaced listview with flatlist [branch ch8329]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8329/mobile-refactor-refactor-and-replace-listview-with-flatlist
#### Testing instructions  
Verify that these 3 refactored list components work and update their data properly:
1. Contact Edit Permissions (files -> "..." options button for a folder -> share -> view shared with)
2. Folder Select (files -> "..." options button for a file -> move)
3. Logs (settings -> help)

Note: there is a bug in Contact Edit Permissions when you try to remove a user. Will edit this PR and post a link to that bug.

Edit: Here is the bug https://app.clubhouse.io/peerio/story/8625/mobile-removing-a-user-from-shared-folder-view-shared-with-tab-visually-removes-the-user-below-too

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
